contributing/community/releases.rst

```diff
@@ -91,7 +91,19 @@ Below is the schedule for the first few versions that use this release model:
 * **Blue** represents the Stabilization phase
 * **Green** represents the Maintenance period
 
-This results in very predictable dates and maintenance periods:
+.. tip::
+
+    If you want to learn more about the timeline of any given Symfony version,
+    use the online `timeline calculator`_.
+
+.. tip::
+
+    Whenever an important event related to Symfony versions happens (a version
+    reaches end of maintenance or a new patch version is released for
+    instance), you can automatically receive an email notification if you
+    subscribed on the `roadmap notification`_ page.
```

Comment:
This links to the same page as the timeline calculator link. Is that intended? What about adding an anchor to the roadmap notificiation box here and to the version checker for the timeline calculator link?